### DISCUSSION TITLE
Combination leader & tracker

### DISCUSSION THREAD

**Discussion by justinfreitag** [nzymedefense/nzyme#491] [Questions] [answered]
Hi, just wondering if this is possible with a single raspberry pi, using one or two wifi adapters?

**Comment by lennartkoopmann** [comment, marked as answer]
This would technically work.

**Reply by justinfreitag** [reply]
Managed to get this working without the LoRa hats. The only source code change was the commenting out the reading of the RSSI value within the existing LoRa device code.
